### PR TITLE
tgz as plain_text

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by antgonza** [comment]
@charles-cowart, this was already deployed in qiita-rc and is working as expected.

File: qtp_biom/validate.py [hunk 1]
```diff
@@ -148,15 +149,18 @@ def validate(qclient, job_id, parameters, out_dir):
...
-            tree = bp.parse_newick(open(phylogeny_fp).read())
-            tree = bp.to_skbio_treenode(tree)
-            filepaths.append((phylogeny_fp, 'plain_text'))
-        except Exception:
-            return False, None, ("Phylogenetic tree cannot be parsed "
-                                 "via scikit-biom")
+        # first let's check if is a tgz, if it is, just pass the file
+        filename = files['plain_text'][0]
+        if is_tarfile(filename):
+            filepaths.append((filename, 'plain_text'))
```
Comment (charles-cowart) [inline qtp_biom/validate.py:155]: Is it okay that we're appending 'plain_text' along with a path to a tarball in this list? It's not clear to me whether the 'plain_text' is a reference to the file's uncompressed contents or the file itself. I'm concerned that the user of this list will have to know to exists() each filename to test if it's a tarball and process accordingly.

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Everything looks good - just need confirmation on one thing.

File: qtp_biom/validate.py (hunk 1 above)
Comment (charles-cowart) [reply qtp_biom/validate.py:155]: Approved. This was discussed in qiita-admin.

**Pull request merged**